src/F2_autoreport.py

```diff
@@ -189,6 +193,31 @@ def make_dash_data_table(df, title=None):
 
 
 def insert_subsection(title, content, **kwargs):
+    r"""
+    Inserts sub-sections within the Dash app layout, such as Input data, simulation results, etc.
+
+    Parameters
+    ----------
+    title : str
+        This is the title or heading of the subsection.
+
+    ...
+
+    content : list
+        This is typically a list of html elements or function calls returning html elements, that make up the
+        body of the sub-section.
+
+     ...
+
+    kwargs : Any possible optional arguments such as styles, etc.
```

Comment:
Maybe list the possible optional arguments in a list?

Reply:
I think the list is in-exhaustive, so its better to leave it as it is

Reply:
You could mention that you only enter examples/a style example. If you are using the function with a kwargs before, that should be easy